app/views/katello/layouts/react.html.erb

```diff
@@ -10,7 +10,8 @@
     <%= notifications %>
     <div id="organization-id" data-id="<%= Organization.current.id if Organization.current %>" ></div>
     <div id="user-id" data-id="<%= User.current.id if User.current %>" ></div>
-    <div id="reactRoot"></div>
-    <%= mount_react_component('katello', '#reactRoot') %>
+    <div id="reactRoot">
+      <%= react_component('katello') %>
```

Comment:
Now that https://github.com/theforeman/foreman/pull/8033 is merged, should these be `foreman_react_component`?

Reply:
update: nevermind, I realized that it's just the web component that was renamed, not the ruby method :)